core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -31,13 +31,20 @@
     GLOBALS.TAG_REGEX = JSON.parse('{{TAG_REGEX|js_string}}');
   </script>
 
-  <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+  <!-- Updated previous version to current version of google charts
+  https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code -->
+  <script type="text/javascript" async src="https://www.gstatic.com/charts/loader.js"></script>
   <script type="text/javascript">
-    if (window.google && window.google.load) {
-      google.load('visualization', '1', {packages: ['corechart']});
-    } else {
-      throw 'error: Could not load google visualization library. Are you offline?';
+    function onloadChartLoadHandler() {
+      if (window.google && window.google.charts) {
+        google.charts.load('current', {packages: ['corechart']});
+      } else {
+        throw 'error: Could not load google visualization library. Are you offline?';
+      }
     }
+    // Since we are using async to load the library wait until entire page is loaded to call google.load().
+    // That should allow the initial page load to complete before starting the chart rendering.
+    window.onload = onloadChartLoadHandler;
```

Comment:
Actually here you should probably use`window.addEventListener('load', ...)`. Otherwise if anything else needs to be fired on load it will get overwritten.

Probably also cleaner to just do it with an anonymous function:

```
    window.addEventListener('load', function() {
      ...
    });
```